src/preloader.js

```diff
@@ -3,6 +3,7 @@
 const generateHeapSnapshot = require('./snapshot')
 const generateHeapSamplingProfile = require('./profile')
 const recordAllocationTimeline = require('./timeline')
+const AbortController = require('abort-controller')
```

Comment:
Should set this up only to use `require('abort-controller')` if the built-in `AbortController` added in 15.0.0 is not present.

Reply:
The added in abort-controller is emitting an experimental warning, I would prefer _not_ to use it.

Reply:
Ok. I'll be opening a PR soon to graduate that from experimental soon.